internal/subshell/sscommon/rcfile.go

```diff
@@ -329,6 +329,12 @@ func SetupProjectRcFile(prj *project.Project, templateName, ext string, env map[
 
 	currExec := osutils.Executable()
 	currExecAbsDir := filepath.Dir(currExec)
+	if bashifyPaths {
+		currExec, err = osutils.BashifyPath(currExec)
```

Comment:
Seeing as we're calling BashifyPath from multiple locations; perhaps it would make sense if the function itself took care of reducing callouts to bash by matching against previous requests.

Reply:
I'm no longer iteratively calling BashifyPath, so this is a non-issue.